samples/style-point-with-distance-composite-scene-symbol/src/main/java/com/esri/arcgismaps/sample/stylepointwithdistancecompositescenesymbol/screens/StylePointWithDistanceCompositeSceneSymbolScreen.kt

```diff
@@ -0,0 +1,90 @@
+/* Copyright 2025 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.stylepointwithdistancecompositescenesymbol.screens
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.toolkit.geoviewcompose.SceneView
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialog
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+import com.esri.arcgismaps.sample.stylepointwithdistancecompositescenesymbol.components.StylePointWithDistanceCompositeSceneSymbolViewModel
+
+/**
+ * Main screen for the sample. Hosts a SceneView and a simple overlay with instructions and distance display.
+ */
+@Composable
+fun StylePointWithDistanceCompositeSceneSymbolScreen(sampleName: String) {
+    val viewModel: StylePointWithDistanceCompositeSceneSymbolViewModel = viewModel()
+
+    // Collect camera distance exposed by the ViewModel.
+    val cameraDistance by viewModel.cameraDistanceMeters.collectAsStateWithLifecycle()
+
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) }
+    ) { padding ->
+        Column(modifier = Modifier.padding(padding)) {
+            // Provide the scene, camera controller, overlays and the SceneViewProxy.
+            SceneView(
+                modifier = Modifier.fillMaxSize().weight(1f),
+                arcGISScene = viewModel.arcGISScene,
+                sceneViewProxy = viewModel.sceneViewProxy,
+                cameraController = viewModel.orbitCameraController,
+                graphicsOverlays = listOf(viewModel.graphicsOverlay),
```

Comment:
Setting the graphicsoverlays like this is not a good idea, since you create a new list instance with every recomposition. You could fix that by using `remember` but since we use a view model anyway, I suggest you expose the collection of graphics overlays on the view model as a property `graphicsOverlays` and then:
```suggestion
                graphicsOverlays = viewModel.graphicsOverlays,
```